README.rst

```diff
@@ -234,6 +234,7 @@ OpenCensus supports integration with popular web frameworks, client libraries an
 Trace Exporter
 --------------
 
+-  `Azure`_
```

Comment:
Do we want the name to be Azure? Or Azure_Monitor? Looks like for consistency it should be Azure_Monitor similar to Stackdriver for GCP. Thoughts?

Reply:
I intend to call it Azure.
This is not Azure Monitor specific, it will include integration with Azure data plane SDKs.